`wallet.dumpPrivKey` and `wallet.importPrivKey` RPC (#306)
DeFiCh/jellyfish: website/docs/jellyfish/api/wallet.md

  CONSERVATIVE = 'CONSERVATIVE'
}
```

## dumpPrivKey

> Reveals the private key corresponding to 'address'. Then the importprivkey can be used with this output.

Reviewer: ```suggestion
Reveals the private key corresponding to an address.
```

```ts title="client.wallet.dumpPrivKey()"
interface wallet {

>   dumpPrivKey(address: string): Promise<string>

Reviewer 2: Add a space between dumpPrivKey and (address: string): Promise<string>

}
```

## importPrivKey

Adds a private key (as returned by dumpprivkey) to your wallet. Requires a new wallet backup.

```ts title="client.wallet.importPrivKey()"
interface wallet {

>   importPrivKey(privkey: string, label: string = "", rescan: boolean = true): Promise<void>

Reviewer 2: Same
